### PR TITLE
Removed unnecessary codes.

### PR DESCRIPTION
**Pull request by nurolopher**
Made come code refactoring, now it's more clean and readable.

### CODE REVIEW

File: Factory.php [hunk 1]
```diff
@@ -25,15 +25,10 @@ public function __construct($phpExcelIO = '\PHPExcel_IOFactory')
      *
      * @return \PHPExcel
      */
-    public function createPHPExcelObject($filename =  null)
+    public function createPHPExcelObject($filename = null)
     {
-        if (null === $filename) {
-            $phpExcelObject = new \PHPExcel();
+        return (null === $filename) ? \PHPExcel() : call_user_func(array($this->phpExcelIO, 'load'), $filename);
```
Comment (liuggio) [inline Factory.php]: missing `new \PHPExcel`

**Comment by liuggio** [comment]
hi @nurolopher thanks for contributing, the tests are red please make it green!! :)

File: Factory.php (hunk 1 above)
Comment (nurolopher) [inline Factory.php]: Thanks! Fixed and now all the tests passed.

**Comment by liuggio** [comment]
@nurolopher GREAT! last thing can you please squash into one commit?

http://symfony.com/doc/2.4/contributing/code/patches.html#rework-your-patch

**Pull request closed without merging**